README.md

```diff
@@ -22,9 +22,36 @@ brew install skycoin-cx
 # Testing changes
 The following process can be used to test changes made to formula:
 1. Ensure the target formula has already been installed on your system. I will use `skycoin-cx` to demonstrate.
-2. Fork this repo on Github and clone it locally to your machine.
+2. Fork the repo on Github and clone it locally to your machine.
 3. Create a new branch for your changes.
 4. Make your changes to the target formula and commit them.
+5. At this point you can choose to either test from your own repo - which is probably the easist, or test using localised insall.
```

Comment:
minor spelling errors

Reply:
Oops. Thx for pointing that out. Will sort that shortly

Reply:
Done. Will submit update soon